controls/mgcp2/src/main/java/org/restcomm/media/control/mgcp/network/netty/MgcpChannelInitializer.java

```diff
@@ -36,21 +36,21 @@
  */
 public class MgcpChannelInitializer extends ChannelInitializer<Channel> {
 
-    private static final int BUFFER_SIZE = 5000;
+    private static int mgcpBufferSize;
     private static final ChannelHandler[] NO_HANDLERS = new ChannelHandler[0];
 
     private final ChannelHandler[] handlers;
 
-    public MgcpChannelInitializer(ChannelHandler... handlers) {
-        this.handlers = (handlers == null || handlers.length == 0) ? NO_HANDLERS : handlers;
+    public MgcpChannelInitializer(int mgcpBufferSize, ChannelHandler... handlers) {
```

Comment:
Please try to see if you can apply the same change without breaking backwards compatibility. 

(this is a public constructor, so by directly changing its signature you're introducing a non-backwards compatible change: anyone using the old constructor must change their code).

Reply:
I have created new constructor to avoid backward compatibility.